README.md

```diff
@@ -100,6 +101,42 @@ General security best practices apply:
 
+# Threat Modeling
+
+The first and most important step for security and privacy is to create a [threat model](https://www.owasp.org/index.php/Application_Threat_Modeling). After all, how can we have security when we don't even know what we're securing against? Each person will have their own needs so everyone's threat model will be different. Threat models tend to evolve over time as our situation changes, so be sure to periodically reassess your threat model.
+
+Step 1: Identify valuable assets
+
+This is probably a lot of things: your phone, your laptop, passwords stored on your devices, internet browsing history, etc. Make a list of all the things you can think of that you want to protect and store it somewhere.
+
+Step 2: Identify adversaries
+
+Define whom you are defending against. Examples could be a nosy roommate, passive data collection on internet infrastructure, a thief, etc. Different threats will have different capabilities, so it's useful to connect which assets you're protecting from which adversaries.
+
+Step 3: Identify the adversaries' capabilities
+
+In order to counter your adversaries, you'll need to understand what they're capable of and what they're not capable of.
+
+Step 4: Come up with ways to mitigate the threats
+
+Now is when you decide the best way to counter each threat. You might avoid writing passwords down on paper so your roommate can't find them or you might encrypt the drive on your computer so a thief can't get data from it.
+
+Step 5: Implement your plan
+
+Start implementing your strategies. Depending on how serious or time-sensitive your situation is, you might want to ease into it so you don't burn yourself out too quickly. Sometimes you might find that a strategy you came up with isn't working for whatever reason, that's ok. Just go back and rethink that part. Eventually you will get to a workable plan that mitigates your threats and that doesn't cause undue inconvenience on your life.
+
+Here's an example of the type of table you should make for each asset you want to protect:
+
+**Asset: Phone**
+
+| Adversary | Motivation | Capabilities | Mitigation |
+| --- | --- | --- | --- |
+| Family | See private chats or browsing history | Close proximity; can see screen or watch me type in my password | Use biometrics, use privacy screen, keep phone locked when I'm not using it |
+| NSA | See contents of packets sent over the internet | Passive surveillance of internet infrastructure, advanced computers for cracking encryption/analysis of packets | Always use encryption for all connections |
+| Thief | Unlock phone and steal personal info and drain bank accounts, sell phone for money | Shoulder surf to see my password, steal my phone out of my hands or when I'm not looking while it's logged in | Keep phone in sight or on my person at all times, keep locked when not in use, use biometrics to avoid typing password in public, use Find My or similar service to track/remotely disable stolen phone |
```

Comment:
This could probably be several types of adversaries, for example a criminal with physical access and another that is remote.

Reply:
I added a "hacker" one